envs/monkey_zoo/blackbox/test_configurations/depth_1_a.py

```diff
@@ -35,6 +36,11 @@ def _add_exploiters(agent_configuration: AgentConfiguration) -> AgentConfigurati
         "Log4ShellExploiter": {},
         "MSSQLExploiter": {},
         "SMB": {"agent_binary_upload_timeout": 30, "smb_connect_timeout": 15},
+        "SNMP": {
+            "agent_binary_download_timeout": 60,
+            "snmp_request_timeout": 0.5,
+            "snmp_retries": 3,
+        },
```

Comment:
Just use the default.

```suggestion
        "SNMP": {
            "snmp_request_timeout": 0.5,
            "snmp_retries": 3,
        },
```

Drop this down to 1 so that our tests don't take forever to run.
```suggestion
        "SNMP": {
            "agent_binary_download_timeout": 60,
            "snmp_request_timeout": 0.5,
            "snmp_retries": 1,
        },
```